lib/ui/tiles/society_tile_left.dart

```diff
@@ -57,12 +57,14 @@ class SocietyTileLeft extends StatelessWidget {
               ),
               Expanded(
                 child: Padding(
-                  padding: const EdgeInsets.all(4.0),
+                  padding: const EdgeInsets.all(0.0),
                   child: Column(
                     crossAxisAlignment: CrossAxisAlignment.start,
                     children: <Widget>[
 
-                      FittedBox(
+                      Container(
+                        width: MediaQuery.of(context).size.width,
+
```

Comment:
`width: MediaQuery.of(context).size.width` is probably not needed because the Row will automatically cover all the given space.
So you can remove the Container Widget completely.

Reply:
Fixed

Reply:
Fixed